src/KingSystem/Resource/resResourceLifeCondition.cpp

```diff
@@ -0,0 +1,88 @@
+#include "KingSystem/Resource/resResourceLifeCondition.h"
+#include <codec/seadHashCRC32.h>
+
+namespace ksys::res {
+
+bool LifeCondition::parse_(u8* data, size_t size, sead::Heap* heap) {
+    parse(data, size, heap);
+    return true;
+}
+
+// NON_MATCHING: two instructions swapped
+bool LifeCondition::parse(u8* data, size_t size, sead::Heap* heap) {
+    if (!data)
+        return true;
+
+    applyResParameterArchive(agl::utl::ResParameterArchive{data});
+    const agl::utl::ResParameterArchive archive{data};
+    const auto root = archive.getRootList();
+
+    const auto invalidWeathers_obj = agl::utl::getResParameterObj(root, "InvalidWeathers");
```

Comment:
Local variables use snake_case:

```suggestion
    const auto invalid_weathers_obj = agl::utl::getResParameterObj(root, "InvalidWeathers");
```